examples/notebooks/Getting Started/Tutorial 4 - Setting parameter values.ipynb

```diff
@@ -0,0 +1,991 @@
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Change individual parameters"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We often want to quickly change a small number of parameter values to investigate how the behaviour or the battery changes. In such cases, we can change parameter values without having to leave the notebook or script you are working in. \n",
+    "\n",
+    "We start initialising the model and the parameter values"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = pybamm.lithium_ion.DFN()\n",
+    "parameter_values = pybamm.ParameterValues(chemistry=pybamm.parameter_sets.Chen2020)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this example we will change the current to 10 A"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "parameter_values[\"Current function [A]\"] = 10"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we just need to run the simulation with the new parameter values"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "1e2bc7fbfaed420f868da0f2f911a8a2",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "interactive(children=(FloatSlider(value=0.0, description='t', max=1690.909090909091, step=16.90909090909091), …"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "sim = pybamm.Simulation(model, parameter_values=parameter_values)\n",
+    "sim.solve()\n",
+    "sim.plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can also simulate drive cycles by passing the data directly"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "parameter_values[\"Current function [A]\"] = \"[current data]US06\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Then we can just define the model and solve it"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "331bef017b4c4970b1a08373292c1527",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "interactive(children=(FloatSlider(value=0.0, description='t', max=600.0, step=6.0), Output()), _dom_classes=('…"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "model = pybamm.lithium_ion.SPMe()\n",
```

Comment:
we should make a mental note to go back and change this once the bug is fixed that prevents reuse of the model in a different sim

Reply:
Agree! Though it shouldn't be a breaking change, right?

Reply:
yeah it will still work, but shouldn't be necessary to create a new model in the future